manual/vcredist2005/vcredist2005.nuspec

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <version>8.0.50727.619501</version>
```

Comment:
This version should be reset to `0.0`, the version is auto-updated by AU but setting the version to the same version as expected will prevent AU to fully run.